docs/DEV-SETUP-WINDOWS.md

```diff
@@ -0,0 +1,81 @@
+## Dev Environment on Windows 10/11 (WSL2)
+
+
+### Install WSL with Ubuntu LTS
+
+Enable WSL 2 on Windows 10/11 for docker support.
+```
+wsl --set-default-version 2
+```
+Install Ubuntu LTS.
+```
+wsl --install Ubuntu
+```
+
+Or follow the instruction here:
+https://learn.microsoft.com/en-us/windows/wsl/install
+
+### Install Docker in windows
+Download the installer from docker and install it.
+
+Check this url for more detailed instructions:
+https://docs.docker.com/desktop/install/windows-install/
+
+You should follow the next steps from within the Ubuntu terminal.
+
+### Install NVM & Node 16
+NVM documentation: https://github.com/nvm-sh/nvm#installing-and-updating
+
+Install NVM
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+```
+Install Node 16
+```
+nvm install 16
```

Comment:
It's worth noting that the officially supported version of node for budibase is `14`. Our docker images are curently based on `node:14-slim`

Reply:
The application may still function completely fine on later versions, but 14 is the version we deploy and test with so there is en element of risk developing with 16+, i.e. features that exist in 16 but not in 14. The safest bet is to develop with the officially supported version.